### PR TITLE
Project: Header with style

### PR DESCRIPTION
**Pull request by Daron976**
## Hello world project 

In this project I:

- Added an h1 header
- Changed the default text colour of the header
- Fixed errors and correct omissions

### CODE REVIEW

**Review by ha-manel — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES ♻️

## Hi @Daron976 👋🏻
Great job so far, There are some changes that need to be implemented in order to proceed to the next project 👍🏻 



## Highlights:  
* All tests passed ✔️
* GitHub flow is followed correctly ✔️
* PR is descriptive and professional ✔️ 

## Required Changes:
- [x] README file should follow Microverse's [template](https://github.com/microverseinc/readme-template). Your README file is lacking many sections, kindly add them (Getting Started, Contributing, Show your support, Acknowledgments...) to make the project more professional ✨ 

## Optional:
- [x] In the future, please refrain from creating a new branch and a new pull request if the reviewer asks for changes. Instead, make the changes in the same old branch and submit the same pull request again 😄 
- [ ] It's better to follow commit messages standards by making them capitalized and in the imperative form (Example: 'Add h1 header' instead of 'added h1 header'). This way, your commit history will be more readable and professional-looking ✨ 

------
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 1]
```diff
@@ -8,19 +8,62 @@
...
 
-## Authors
+### Deployment
+- Right click on the index.html file
+- Select "Open with" / "Open with another application"
+- Choose any browser from the options that appear
+
+## Author/s
 
 👤 **Author1**
```
Comment (Kamba56) [inline README.md:45]: - [ ] [Optional] You can add your name here instead of 'Author'

File: README.md [hunk 2]
```diff
@@ -8,19 +8,62 @@
...
 
+## Getting Started
+
+**Follow these instruction below to setup a this project in your local environment**
+
 ### Prerequisites
 
 - Html & CSS basics
 - Git & github
+- SSH key for github linked to git on local environment
```
Comment (Kamba56) [inline README.md:19]: - [ ] [Optional] I think prerequisites of a project are things that you need to install locally before the project will run effectively locally. In that case, the prerequisites of this project should be `node js` and `npm`.😊

**Review by Kamba56 — APPROVED** [review]
Hi @Daron976 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Daron976** [comment]
Thank you so much

I wanted to change the commit message but I couldn’t alter it
In future I’ll definitely use better commit messages

**Pull request merged**